fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -2468,6 +2485,79 @@ module.exports = function(config, DB) {
       })
     })
 
+    describe('change email', () => {
+      let account, secondEmail
+
+      before(() => {
+        account = createAccount()
+        account.emailVerified = true
+        secondEmail = createEmail({
+          uid: account.uid,
+          isVerified: true
+        })
+        return db.createAccount(account.uid, account)
+          .then(function () {
+            return db.createEmail(account.uid, secondEmail)
+          })
+          .then(function (result) {
+            assert.deepEqual(result, {}, 'Returned an empty object on email creation')
+            return db.accountEmails(account.uid)
+          })
+          .then(function (res) {
+            assert.deepEqual(res.length, 2, 'Returns correct amount of emails')
+            assert.equal(res[0].email, account.email, 'primary email is the address that was used to create account')
+            assert.deepEqual(res[0].emailCode, account.emailCode, 'correct emailCode')
+            assert.equal(!! res[0].isVerified, true, 'correct verification set')
+            assert.equal(!! res[0].isPrimary, true, 'isPrimary is true')
+
+            assert.equal(res[1].email, secondEmail.email, 'primary email is the address that was used to create account')
+            assert.deepEqual(res[1].emailCode, secondEmail.emailCode, 'correct emailCode')
+            assert.equal(!! res[1].isVerified, true, 'correct verification set')
+            assert.equal(!! res[1].isPrimary, false, 'isPrimary is false')
```

Comment:
> Does it report everything as clearly as a separate it block would?

Yea it does. I tend to like the asserts like this because I normally run a few tests at a time and it helps verify that I am in a known state since I wouldn't have necessarily run through the entire account create tests.